app/controllers/follower_accounts_controller.rb

```diff
@@ -2,7 +2,9 @@
 
 class FollowerAccountsController < ApplicationController
   include AccountControllerConcern
+  include SignatureVerification
 
+  before_action :require_signature!, if: -> { request.format == :json && authorized_fetch_mode? }
```

Comment:
feels deeply weird to me that we would allow people to scrape this info from the html but refuse to show it in a machine-readable format.

Reply:
Stuff doesn't federate over HTML pages.

Reply:
There's no way to have people able to access HTML pages without auth and also prevent scraping. Mastodon already supports not showing followers publicly, and unless federated followers lists becomes a thing in Mastodon, the HTML won't need a separate setting.